### ISSUE TITLE
fft3w-deploy: MPI hello world example doesn't build

### ISSUE THREAD

**Issue by brucellino** [SouthAfricaDigitalScience/fftw3-deploy#2]
Full error output: 

```
hello-world-mpi.cpp: In function 'int main(int, char**)':
hello-world-mpi.cpp:2: error: 'ptrdiff_t' does not name a type
hello-world-mpi.cpp:3: error: 'fftw_plan' was not declared in this scope
hello-world-mpi.cpp:3: error: expected ';' before 'plan'
hello-world-mpi.cpp:4: error: 'fftw_complex' was not declared in this scope
hello-world-mpi.cpp:4: error: 'data' was not declared in this scope
hello-world-mpi.cpp:5: error: 'ptrdiff_t' was not declared in this scope
hello-world-mpi.cpp:5: error: expected ';' before 'alloc_local'
hello-world-mpi.cpp:7: error: 'MPI_Init' was not declared in this scope
hello-world-mpi.cpp:8: error: 'fftw_mpi_init' was not declared in this scope
hello-world-mpi.cpp:11: error: 'alloc_local' was not declared in this scope
hello-world-mpi.cpp:11: error: 'N0' was not declared in this scope
hello-world-mpi.cpp:11: error: 'N1' was not declared in this scope
hello-world-mpi.cpp:11: error: 'MPI_COMM_WORLD' was not declared in this scope
hello-world-mpi.cpp:12: error: 'local_n0' was not declared in this scope
hello-world-mpi.cpp:12: error: 'local_0_start' was not declared in this scope
hello-world-mpi.cpp:12: error: 'fftw_mpi_local_size_2d' was not declared in this scope
hello-world-mpi.cpp:13: error: 'fftw_alloc_complex' was not declared in this scope
hello-world-mpi.cpp:16: error: 'plan' was not declared in this scope
hello-world-mpi.cpp:17: error: 'FFTW_FORWARD' was not declared in this scope
hello-world-mpi.cpp:17: error: 'FFTW_ESTIMATE' was not declared in this scope
hello-world-mpi.cpp:17: error: 'fftw_mpi_plan_dft_2d' was not declared in this scope
hello-world-mpi.cpp:20: error: 'i' was not declared in this scope
hello-world-mpi.cpp:20: error: 'j' was not declared in this scope
hello-world-mpi.cpp:21: error: 'my_function' was not declared in this scope
hello-world-mpi.cpp:24: error: 'fftw_execute' was not declared in this scope
hello-world-mpi.cpp:26: error: 'fftw_destroy_plan' was not declared in this scope
hello-world-mpi.cpp:28: error: 'MPI_Finalize' was not declared in this scope
--------------------------------------------------------------------------
mpirun was unable to launch the specified application as it could not access
or execute an executable:

Executable: ./hello-world-mpi
Node: sl6slave
```

**Comment by brucellino** [comment]
ok, the code in the file was simply incomplete :-/

**Comment by brucellino** [comment]
having changed the code a bit, we are now getting: 
`‘fftw_mpi_init’ was not declared in this scope`

**Comment by brucellino** [comment]
fixed in d62ea5e9618
